packages/proxy/lib/http/response-middleware.ts

```diff
@@ -492,6 +499,10 @@ const SendResponseBodyToClient: ResponseMiddleware = function () {
   this.res.on('end', () => this.end())
 }
 
+const CleanupCypressHeaders: ResponseMiddleware = function () {
+  delete this.req.headers['x-cypress-is-aut-frame']
+}
```

Comment:
this seems like it should happen in `request-middleware` before the request is sent, it seems like we are still sending the request outbound with the `x-cypress-is-aut-frame` header. this could use a test.

Reply:
Addressed in https://github.com/cypress-io/cypress/pull/19323/commits/127f8936dea5221a0e0e26f58140395ed2b9f6bd